pkgs/top-level/all-packages.nix

```
@@ -23581,10 +23581,14 @@ in
 
   kssd = callPackage ../applications/science/biology/kssd { };
 
+  last = callPackage ../applications/science/biology/last { };
+
   macse = callPackage ../applications/science/biology/macse { };
 
   migrate = callPackage ../applications/science/biology/migrate { };
 
+  minia = callPackage ../applications/science/biology/minia { };
```

may be good to use a compatible range of boost, in case the default gets changed to something like boost_171
```suggestion
  minia = callPackage ../applications/science/biology/minia {
    boost = boost16x;
  };
```